scripts/get_continent_map.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3.7
```

Comment:
Thats not working for every one. Your distro should provide a symlink with the name `python` to the installed python version.